istio.deps

```diff
@@ -4,7 +4,7 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "08cec447dfc349b314e226dc66132dbdb5d08fc9"
+    "lastStableSHA": "586ca4896aac3c3cad7e6596afddbc89e376a43e"
```

Comment:
It seems there's a new commit, which also updates Envoy, do you want to include it as well?